Socket/env.js

```diff
@@ -6,11 +6,13 @@ const VITE_MESSAGING_SENDER_ID="195506194018";
 const VITE_APP_ID="1:195506194018:web:6c1f8301b9d0db195b333b";
 const VITE_MEASUREMENT_ID="G-QTKCH75V4J";
 const VITE_VAPID_KEY="BIk2IwWnyh6QKW0Y6c4ZCNLuNltnCcaGT4hxKNdJl6wgpvQhYfDIkZZ34iU14Ox5zSflGAPL5dmDRbv7GENQrqA"
+const GOOGLE_PLACES_KEY = "AIzaSyDiMeYprJDeYoM-WclTeU2mHqS4PfEJmaY"
```

Comment:
Don't commit API key in commits.

Reply:
I included the file name in my .gitignore, I can't point out why git still commits it